packaging/dashboard.version

```diff
@@ -0,0 +1 @@
+v0.2.4
```

Comment:
We already released the version 0.3.1 with some bug fixes ( https://github.com/netdata/dashboard/releases) , please, could you update for the newest?

Reply:
Thanks, I hadn't noticed this yet. I'll make sure it's up-to-date before marking this as ready-fo-review.